### DISCUSSION TITLE
Sort scales

### DISCUSSION THREAD

**Discussion by pavelkeyzik** [primer/prism#17] [Ideas]
At the moment we're not able to sort scales and I think it might be useful to sort them in two ways:
1. by dragging;
2. alphabetically;

![CleanShot 2022-06-15 at 11 32 00@2x](https://user-images.githubusercontent.com/17102399/173781877-7e53c3ee-c676-446c-aed6-d9dbcbb9d3b7.png)

**Comment by colebemis** [comment]
Great idea. This would be very handy ✨
